draft-ietf-masque-h3-datagram.md

```diff
@@ -359,23 +359,35 @@ violation of these requirements MUST treat the HTTP message as malformed.
 
 ## Error Handling
 
-When an error occurs in processing the Capsule Protocol, the receiver MUST treat
-the message as malformed or incomplete, according to the underlying transport
-protocol. For HTTP/3, the handling of malformed messages is described in
-{{Section 4.1.3 of H3}}. For HTTP/2, the handling of malformed messages is
+Unless otherwise stated, Capsule Protocol endpoints are expected to apply the
+same message format rules as the underlying HTTP version. When a receiver
+encounters a processing error, it MUST be treated as if it were a malformed or
+incomplete message. For HTTP/3, the handling of malformed messages is described
```

Comment:
I'm not sure this first sentence adds much value, because it sounds to me like "please follow referenced specs unless told not to"

Also I'd specifically call out the capsule protocol in the normative sentence that follows

```suggestion
When a receiver encounters an error processing the Capsule Protocol, the receiver
MUST treat it as if it had received a malformed or incomplete HTTP message.
For HTTP/3, the handling of malformed messages is described
```

Reply:
With the change below, this suggestion is better, thanks.